src/tab/Tabs.tsx

```diff
@@ -26,21 +57,9 @@ export const Tabs: React.FC<TabsProps> = ({
     <MuiTabs
       value={value}
       onChange={onChange}
-      // css={[
-      //   css`
-      //     &.MuiTabs-root {
-      //       ${reverse
-      //         ? tw`(border-shade-light-default border-t-2)!`
-      //         : tw`(border-shade-light-default border-b-2)!`}
-      //     }
-      //     & .MuiTabs-indicator {
-      //       ${colorStyle[color]}
-      //       ${reverse && tw`top-0`}
-      //     }
-      //   `,
-      //   noBorder && tw`border-none!`,
-      //   twin,
-      // ]}
+      classes={{
```

Comment:
min-heightの設定の修正をお願いします

```suggestion
      classes={{
        root: clsx(['min-h-[auto]']),
```